Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -134,6 +150,37 @@ namespace AzQtComponents
         return line;
     }
 
+    QSize BreadCrumbs::sizeHint() const
+    {
+        const QFontMetrics fm(m_label->font());
+        const qreal separatorWidth = fm.horizontalAdvance(g_plainTextSeparator);
+        // Icon adds the icon itself plus two spaces, see generateIconHtml()
+        const qreal iconWidth = g_iconWidth + fm.horizontalAdvance("\u00a0\u00a0");
+
+        // TODO(Qt 5.15.2): replace with:
```

Comment:
Does this mean replace once we move to a version >= 5.15.2? Our current binary package is incorrectly labeled as 5.15.2, but it's actually a branch of 5.15.1:

https://github.com/o3de/qt5/tree/5.15.1-o3de

Reply:
Yes. I'll create a separate task for it.

Reply:
https://github.com/o3de/o3de/issues/13336 - task is created. However, it would probably be good to have a tag that will denote todos to be done when the move to the actual 5.15.2 is made.

Reply:
The Qt documentation states that `QStringView::split` was [introduced in Qt 6.0](https://doc.qt.io/qt-6/qstringview.html#split:~:text=be%20dangling%20pointers.-,This%20function%20was%20introduced%20in%20Qt%206.0.,-template%20%3Ctypename%20Container). So I think this can't be done until we move to Qt 6.